apps/client/src/components/Settings/AccountSettings.svelte

```diff
@@ -42,16 +66,16 @@
     <p class="text-neutral">Connect your social accounts.</p>
     <div class="flex gap-2">
       <Button
-        CTA="Connect GitHub"
-        icon="ri:github-line"
-        onClick={connectGithub}
-        bg="base-200"
+        CTA={buttons[0].CTA}
```

Comment:
I think you can now iterate over this rather than accessing every element via and index.